lua/read_file.lua

```diff
@@ -0,0 +1,10 @@
+-- Lua script to read input, transform to uppercase, and print with line numbers
+local file_path = arg[1]
+local fh = io.open(arg[1], "r")
```

Comment:
```suggestion
local fh = io.open(file_path, "r")
```

(wasn't using the variable from the line above)